### PR TITLE
Update/genesis blocks

### PR DESCRIPTION
**Pull request by BMO-tech**
**IN DRAFT UNTIL A RELEASE CANDIDATE FOR GENESIS BLOCKS IS AVAILABLE TO TEST WITH.**

Updates the theme to use Genesis Blocks instead of Atomic Blocks.

**How to test**
Code review only

**Documentation**
Customer Facing Documentation required before release cc @susannelson

**Suggested changelog entry**
- Included in PR

= 3.4.0 =

Genesis Sample 3.4.0 requires the Genesis Blocks plugin.

* Changed: Install Genesis Blocks instead of Atomic Blocks during one-click theme setup.
* Changed: Updated one-click-theme-setup content to use Genesis Blocks.
* Changed: Updated Editor and front-end styles for Genesis Blocks compatibility.

### CODE REVIEW

File: config/onboarding-shared.php [hunk 1]
```diff
@@ -16,9 +16,9 @@
 return [
 	'plugins'          => [
 		[
-			'name'       => __( 'Atomic Blocks', 'genesis-sample' ),
-			'slug'       => 'atomic-blocks/atomicblocks.php',
-			'public_url' => 'https://atomicblocks.com/',
+			'name'       => __( 'Genesis Blocks', 'genesis-sample' ),
+			'slug'       => 'genesis-blocks/genesisblocks.php',
```
Comment (mindctrl) [inline config/onboarding-shared.php:20]: The plugin filename in GB is `genesis-blocks.php`

File: lib/gutenberg/front-end.css [hunk 2]
```diff
@@ -13,7 +13,8 @@
 /* Paragraphs
 ---------------------------------------------------------------------------- */
 
-.alignfull:not(.has-background) .wp-block-group__inner-container > p:not(.has-background-color) {
+.alignfull:not(.has-background)
+.wp-block-group__inner-container > p:not(.has-background-color) {
```
Comment (dreamwhisper) [inline lib/gutenberg/front-end.css:17]: This should all be on one line.

File: lib/gutenberg/front-end.css [hunk 3]
```diff
@@ -209,7 +209,8 @@
 	max-width: 80%;
 }
 
-.site-container .wp-block-pullquote:not(.is-style-solid-color)[style*="border-color"] {
+.site-container
+.wp-block-pullquote:not(.is-style-solid-color)[style*="border-color"] {
```
Comment (dreamwhisper) [inline lib/gutenberg/front-end.css:213]: This should be on one line.

File: lib/gutenberg/front-end.css (hunk 2 above)
Comment (dreamwhisper) [inline lib/gutenberg/front-end.css:17]: This should be on one line.

File: lib/gutenberg/front-end.css [hunk 4]
```diff
@@ -373,15 +373,13 @@ hr.wp-block-separator {
 		margin-right: calc(-100vw / 2 + 100% / 2);
 		width: 100vw;
 	}
-
```
Comment (dreamwhisper) [inline lib/gutenberg/front-end.css:376]: Not a huge deal, but we usually keep these spaces for CSS so spacing is even before and after the rule.

Comment (BMO-tech) [reply lib/gutenberg/front-end.css:376]: hmm, that looks like a `stylelint` auto-fix

Comment (BMO-tech) [reply lib/gutenberg/front-end.css]: So, in VS Code, when I added the extra space it was auto-tabbed to start writing more CSS and style lint complained about "No whitespace at EOL" or something like that. Simply removing the auto-tab prevent the auto-fix from messing things up.

:laughing:

**Comment by johnstonphilip** [comment]
Working for me!

**Comment by johnstonphilip** [comment]
@dreamwhisper I had to resolve a merge conflict when merging develop in again. Can you give this one more look (don't need to test).

**Comment by dreamwhisper** [comment]
@johnstonphilip Looks good. I figured you may get a merge conflict this time because there was an interim release. The only thing we should do is update the shortened URL for Genesis Blocks to https://wordpress.org/plugins/genesis-blocks/ (shortened URL works, but this is more direct).

**Comment by BMO-tech** [comment]
Looking good, I give it a stamp of approval!

**Pull request merged**